### PR TITLE
Solutions to Django Common Errors in Web Development

### LINKED ISSUE #5922: Solutions to Common Django Errors in Web Development
**Suggested article title**

Solutions to Common Django Errors in Web Development

**Proposed topic introduction**

Many times, people get stuck on the way, not only on coding related development but also several other development. This may be as a result of failure to follow instructions, installation error, and various other major or minor reasons. In this article, we shall major on Django errors during web development.
It is almost impossible to reach your destination in web development using any other framework including django without getting stuck on the way. At times there must be an emergence of error. Considering web development, it can be as a result of wrong path, installation, procedures and just ignorance. In this article, I'll make it easy for youby clearly pointing to some common errors associated with django, explaining to you step by step how we can solve these errors to keep going and reach our destination. Happy reading.

**Key takeways**

- 404 page not found error
- No reverse match error
- No module named error
- Template does not exist error
- Djanho-admin not recognized as internal and external command
- Djangi not able to find static files
- NameError
- Internal server error

**Article quality**

In my article, I'll be solving different errors in one article making it easier for the reader to easily find the error and solve it. Not as other articles, in my article, I will be considering several causes of a particular error and give solutions to them since an error can be brought about by many inconveniences. The aim of my article is to make it easy for programmers to reach their destination.

**References**

N/A

### PR DESCRIPTION
**Pull request by OkothAlvan**
`##` Article checklist

## Our **Peer Reviewer and Content Moderator teams do NOT provide any revisions services.** All revisions and edits should be completed on your own forked repo (as to not take up room in the queue) in order for our team to review them all in a timely manner. 

**ANY ARTICLE SUBMITTED WITH GLARING ERRORS WILL BE IMMEDIATELY CLOSED.**

>As a rule of thumb - please be sure to only submit articles (pull requests) that are fully polished and ready to be published. Be sure to go through our resources documents for extra and 3rd party (vetted) resources to help improve overall technical writing.

> If you have not already, please go over our [Resources Page](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/resources-page.md) for more tips and tricks on how to improve your overall technical writing so reviews are swifter, increase the chance of a payout and provide more value to our readers.

> For first-time contributors and for more details on our submission guidelines, **see our [Contributing Guidelines](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/CONTRIBUTING.md)**.

#### Formatting and structure checks
- [x] Have you used the correct folder and file structure? - **see [Contribution Guidelines](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/CONTRIBUTING.md)**
- [x] Is your article properly formatted in Markdown? - **see [Github Markdown guide](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)**
- [x] Have you used our preferred heading structure? - **[H3 (###) and up](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#headers)**
- [x] Does your article follow any of the suggested structure formats? - **see [example formats](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md#example-format-structure)**

- [x] Is your article over 750 words? - **Tool**: [Wordcounter.net](https://wordcounter.net)
- [x] Is the article you are submitting an in-depth and unique article? Does it go beyond what is in the official docs and what is covered in other blog sites. See these articles as [examples](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/resources-page.md).
- [x] Does your article provide enough value and detail about your topic? - **Articles should be clear, accurate, and fully explained.**
- [x] Can your article be understood by beginners? - **Assume the audience is smart but has no prior exposure to the common terminology in your article.**

- [x] Have you included a hero image that is **600x400 pixels**, under **300KB in size**, and have the copyright to use it? - **Tools:** [Pixlr Image Editor](https://pixlr.com/e) for resizing and [Unsplash](https://unsplash.com) for Creative Commons images
      
#### Grammar & plagiarism checks
- [x] Have you spellchecked and grammar checked your article? - **Tools:** [Write&Improve](https://writeandimprove.com/?lang=en-GB) - [ProWritingAid](https://prowritingaid.com/) - [Grammarly](https://grammarly.com) 
- [x] Please place your article through a 3rd party plagiarism checker. We suggest using Quetext, [this tool](https://www.quetext.com/) is free and has a daily limit. **We typically accept articles with 10% or less.**
- [x] Have you checked your article for readability? - **Tool:** [Hemmingway](http://www.hemingwayapp.com)
- [x] Have you added sources for quotes and images that aren't yours?

#### Technical checks
- [x] Are your code snippets properly formatted for syntax highlighting - **see [Syntax guide](https://gohugo.io/content-management/syntax-highlighting)**
- [x] Have you checked your code runs correctly and you've highlighted all necessary dependencies for installation?
- [x] Are the software programs and packages you're highlighting in your article up to date, using current versions and not deprecated?

This closes #5922

> Finally, delete the article checklist notes in blockquotes and submit your PR. We look forward to reviewing your article.

### CODE REVIEW

**Comment by srishilesh** [comment]
@OkothAlvan 

Welcome to EngEd community!

Your PR has entered our reviewing queue. Please wait until a peer-reviewer pings you to get started with the review process.

Thanks.

**Review by Jethro-magaji — CHANGES_REQUESTED** [review]
Hi @OkothAlvan 
Well done for such an amazing article. However, there're a lot of corrections you need to make.
Please, run your article on tools like Grammarly and ProWritingAid for better editing.
We will be closing this PR to allow you to polish the article.
Visit the EngEd new contributor's docs [here](https://github.com/section-engineering-education/engineering-education/tree/master/new_contributors) to guide you.
Thanks.

cc @section-engineering-education/content-moderator-team please close this PR

File: content/articles/solutions-to-django-common-errors-in-web-development/index.md [hunk 1]
```diff
@@ -0,0 +1,151 @@
+# **Solutions to Common Django Errors in Web Development** 
+
+## **Introduction**
```
[the remark this answers is not in the document]
Comment (OkothAlvan) [reply content/articles/solutions-to-django-common-errors-in-web-development/index.md]: Hi @Jethro-magaji 
What happens after I will be done with the revisions if the PR is closed?

**Comment by srishilesh** [comment]
> Hi @OkothAlvan Well done for such an amazing article. However, there're a lot of corrections you need to make. Please, run your article on tools like Grammarly and ProWritingAid for better editing. We will be closing this PR to allow you to polish the article. Visit the EngEd new contributor's docs [here](https://github.com/section-engineering-education/engineering-education/tree/master/new_contributors) to guide you. Thanks.
> 
> cc @section-engineering-education/content-moderator-team please close this PR

@Jethro-magaji 

You may point out all the corrections to be made, and close the PR by yourself.

**Comment by OkothAlvan** [comment]
Noted. Thank you.

**Pull request closed without merging**